umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10.clj

```diff
@@ -317,6 +317,10 @@
    :TilingIdentificationSystems (spatial/parse-tiling doc)
    :ProcessingLevel {:Id (su/with-default (value-of doc "/DIF/Product_Level_Id") sanitize?)}
    :AdditionalAttributes (aa/xml-elem->AdditionalAttributes doc sanitize?)
+   :StandardProduct (last (for [metadata (select doc "/DIF/Extended_Metadata/Metadata")
```

Comment:
so if you put in more then one, your going to ignore all but the last? Also will this still work if there is one ExtendedMetadata with several Metadata tags instead of multiple ExtendedMetadata tags?

Reply:
Yes, it works.